### PR TITLE
Cleanup of outdated composer files and vendor directory

### PR DESCRIPTION
**Pull request by wshafer**
This should give us a new base for the OS skeleton.  Please double check the composer.json file and make sure I have all the required components.

### CODE REVIEW

**Comment by rodmcnew** [comment]
We couldn't see the whole PR diff due to "Sorry, we could not display the entire diff because too many files (3,664) changed."

We deleted then vender folder which should help. Could you merge and push again to get all vendor folder changes out of the diff?

**Comment by rodmcnew** [comment]
I am going to try to close and re-open this. If I have access to re-open, this may fix the dif.

**Comment by rodmcnew** [comment]
Hmm.. close and reopen didn't work. Westin please try a merge and push.

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 .idea
```
Comment (jerv13) [inline .gitignore]: We should leave the data folder in.

**Pull request merged**